src/e2e/java/teammates/e2e/cases/StudentProfilePageE2ETest.java

```diff
@@ -42,6 +42,27 @@ public void testAll() {
         profilePage.editProfilePhoto();
         profilePage.verifyPhotoSize("220px", "220px");
 
+        ______TS("Typical case: Profile picture ratios");
+        profilePage.fillProfilePic("src/test/resources/images/profile_pic_too_wide.jpg");
```

Comment:
The lines from `profilePage.fillProfilePic` to `profilePage.waitForUploadEditModalVisible` are repeatable 3 times. They should be abstracted out.

Reply:
I understand, will also be changed